vscode-powershell.build.ps1

```diff
@@ -101,13 +101,8 @@ task BuildAll BuildEditorServices, Build
 #region Test tasks
 
 task Test Build, {
-    if (!$global:IsLinux) {
-        Write-Host "`n### Running extension tests" -ForegroundColor Green
-        exec { & npm run test }
-    }
-    else {
-        Write-Warning "Skipping extension tests on Linux platform because vscode does not support it."
-    }
+    Write-Host "`n### Running extension tests" -ForegroundColor Green
```

Comment:
```suggestion
    if ($env:TF_BUILD -and $global:IsLinux) {
        Write-Warning "Skipping extension tests in Linux CI because vscode does not support it."
        return
    }
    Write-Host "`n### Running extension tests" -ForegroundColor Green
```